duckduckgo/duckduckgo-privacy-extension
# ATB parameter in MV3

**Reviewer:**



## Description:

Adds ATB to search URL. 

## Steps to test this PR:
1. From a new Chrome profile, install the extension
2. Do a search from URL bar, atb should be added to the URL
3. Check other DDG pages, they should not have atb added to them



###### Reviewer Checklist:


###### PR Author Checklist:
